flink-table/flink-sql-gateway-api/src/main/java/org/apache/flink/table/gateway/api/SqlGatewayService.java

```diff
@@ -198,4 +198,15 @@ ResultSet fetchResults(
      * @return names of the registered catalogs.
      */
     Set<String> listCatalogs(SessionHandle sessionHandle) throws SqlGatewayException;
+
+    /**
+     * Return all available schemas' name matching {@param databaseNamePattern} in the given
```

Comment:
We don't allow users to specify databaseNamePattern here.

Reply:
Fixed.